test/system/product/unchecked_test.rb

```diff
@@ -9,22 +9,37 @@ class ProductsTest < ApplicationSystemTestCase
   end
 
   test 'advisor can not see listing unchecked products' do
-    visit_with_auth '/products', 'advijirou'
-    assert_no_link '未チェック'
+    login_user 'advijirou', 'testtest'
+    visit '/products'
```

Comment:
```suggestion
    visit_with_auth '/products', 'advijirou'
```

すみません、この辺りもこっちでお願いします〜